common/SmokeTests/SmokeTest/Update-Dependencies.ps1

```diff
@@ -67,6 +76,7 @@ Log-Info "Querying package information."
 $referenceUpdateCount = 0
 
 Get-SmokeTestPkgProperties -ArtifactsPath $ArtifactsPath
+| Where-Object { -not $packageExcludeSet.Contains($_.Name) }
```

Comment:
Calling a .NET API like this will use the default comparison, which is almost undoubtedly case-sensitive. Instead, use:

`where-object $packageExcludeSet -notcontains Name`